packages/css/src/list.css

```
@@ -1,17 +1,17 @@
 .ds-list {
   --dsc-list-padding-left: var(--ds-size-6);
-  --dsc-list-spacing: var(--ds-size-3);
-  --dsc-list-spacing-nested: var(--ds-size-2);
+  --dsc-list-margin-top: var(--ds-size-3);
```

```suggestion
  --dsc-list-content-spacing: var(--ds-size-3);
  --dsc-list-nested-spacing: var(--ds-size-3);
```
..samme som på card? ☺️

---

Ueinig, det er ingen "content" her, og som er skriver i beskrivelsen på PRen, så er ordet `spacing` der det ikkje mapper 1:1 på ein css prop